python/ray/tune/trial.py

```diff
@@ -372,7 +386,22 @@ def update_resources(self, cpu, gpu, **kwargs):
         """
         if self.status is Trial.RUNNING:
             raise ValueError("Cannot update resources while Trial is running.")
-        self.resources = Resources(cpu, gpu, **kwargs)
+        if isinstance(resources, PlacementGroupFactory):
+            self.resources = Resources(cpu=1, gpu=0)
+            self.placement_group_factory = resources
+        elif callable(resources):
+            self.resources = Resources(cpu=1, gpu=0)
```

Comment:
do we need to adjust this?

Reply:
Not after the last commit. Thanks for catching, remove